web/src/components/DropdownMenuItem.tsx

```diff
@@ -22,21 +22,22 @@ import theme from '../theme';
 import { MenuItem as IMenuItemProps } from '../types';
 
 const StyledDropdownItem = styled.div`
-  margin: 15px 15px 15px 0px;
+  margin: 20px 15px 15px 0px;
   border-bottom: 1px solid;
   @media (max-width: ${theme.breakpoints[1]}) {
     color: ${theme.colors.contrast};
   }
 `;
 
 const DropdownMenuItem: React.FC<IMenuItemProps> = (props) => {
-  const { href, title, subTitle, onClickCB } = props;
+  const { href, title, subTitle, onClickCB, handleOnClick } = props;
+
   if (!!href) {
```

Comment:
I think `if (!!href)` is the same as `if (href)` no?

Reply:
Yes...ish. !! just guarantees a bool via coercion.